components/automate-cli/cmd/chef-automate/certRotate_test.go

```diff
@@ -775,70 +849,71 @@ func TestGetFrontEndIpsForSkippingCnAndRootCaPatching(t *testing.T) {
-			skipIpsList: c.getIps("frontend", infra),
+			infra:               infra,
+			skipIps:             true,
 		},
 		{
-			testCaseDescription: "Opensearch root-ca and cn patching | same root-ca different cn",
+			testCaseDescription: "Comparing old cn nd old rootCA | same root-ca different cn",
 			newRootCA:           FileContent,
+			oldRootCA:           FileContent,
 			newCn:               newCn + "n",
 			oldCn:               "",
-			currentCertsInfo: &certShowCertificates{
-				OpensearchRootCert: FileContent,
-			},
-			infra:       infra,
-			skipIpsList: []string{},
+			infra:               infra,
+			skipIps:             false,
 		},
 		{
-			testCaseDescription: "Opensearch root-ca and cn patching | same cn and node flag",
+			testCaseDescription: "Comparing old cn nd old rootCA | different root-ca",
+			newRootCA:           FileContent + "a",
+			oldRootCA:           FileContent,
 			newCn:               newCn,
 			oldCn:               newCn,
-			node:                ValidIP4,
-			currentCertsInfo:    &certShowCertificates{},
 			infra:               infra,
-			skipIpsList:         c.getIps("frontend", infra),
+			skipIps:             false,
 		},
 		{
-			testCaseDescription: "Opensearch root-ca and cn patching | different cn and node flag",
-			newCn:               newCn + "n",
+			testCaseDescription: "Comparing old cn nd old rootCA | different root-ca same cn",
+			newRootCA:           FileContent + "a",
+			oldRootCA:           FileContent,
+			newCn:               newCn,
 			oldCn:               newCn,
-			node:                ValidIP4,
-			currentCertsInfo:    &certShowCertificates{},
 			infra:               infra,
-			skipIpsList:         []string{},
+			skipIps:             false,
 		},
 		{
-			testCaseDescription: "Opensearch root-ca and cn patching | different root-ca",
-			newRootCA:           FileContent + "a",
+			testCaseDescription: "Comparing old cn | different cn with node flag",
+			newCn:               newCn + "a",
+			oldCn:               newCn,
+			infra:               infra,
+			node:                ValidIP,
+			skipIps:             false,
+		},
+		{
+			testCaseDescription: "Comparing old cn | same cn with node flag",
 			newCn:               newCn,
 			oldCn:               newCn,
-			currentCertsInfo: &certShowCertificates{
-				OpensearchRootCert: FileContent,
-			},
-			infra:       infra,
-			skipIpsList: []string{},
+			infra:               infra,
+			node:                ValidIP,
+			skipIps:             true,
 		},
 	}
 
 	for _, testCase := range testCases {
 		t.Run(testCase.testCaseDescription, func(t *testing.T) {
-			skipIpsListGot := c.getFrontEndIpsForSkippingCnAndRootCaPatching(testCase.newRootCA, testCase.newCn, testCase.oldCn, testCase.node, testCase.currentCertsInfo, infra)
-			assert.Equal(t, testCase.skipIpsList, skipIpsListGot)
+			skipIpsGot := c.getFrontEndIpsForSkippingCnAndRootCaPatching(testCase.newRootCA, testCase.newCn, testCase.oldCn, testCase.oldRootCA, testCase.node, infra)
```

Comment:
Give meaning full name to this skipIpsGot variable also. Ex. isIpSkipped

Reply:
done